simapp/simd/cmd/root.go

```diff
@@ -2,6 +2,7 @@ package cmd
 
 import (
 	"errors"
+
```

Comment:
Why did you add empty line here?

Reply:
No meaning. I think this line was added automatically by IDE.